bucket/eclipse-sumo.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "1.7.0",
+    "description": "Eclipse SUMO is an open source, microscopic and continuous traffic simulation package designed to handle large networks.",
+    "homepage": "https://www.eclipse.org/sumo/",
+    "license": "Eclipse Public License Version 2",
+    "architecture": {
+        "64bit": {
+            "url": "https://sumo.dlr.de/releases/1.7.0/sumo-win64-1.7.0.zip",
+            "hash": "9691366c44e1b3518b561643aa176ddfc12514037a78f01195f07ba6fe73ca9e"
+        },
+        "32bit": {
+            "url": "https://sumo.dlr.de/releases/1.7.0/sumo-win32-1.7.0.zip",
+            "hash": "23635d78bec7ed2747f3b7705bf2636028ffa5b5f8190ed3e39f6e484e4bab60"
+        }
+    },
+    "bin": [
+        "bin\\sumo-gui.exe",
+        "bin\\netedit.exe"
+    ],
+    "shortcuts": [
+        [
+            "sumo-gui.exe",
+            "SUMO - GUI"
+        ]
+    ],
+    "env_add_path": "bin",
+    "env_set": {
+        "SUMO": "$dir"
+    },
```

Comment:
Why??

Reply:
This is because the command line tools in the folder "bin" need to be used by users. There are many python scripts in the folder "tools" requres the environment "SUMO_HOME". See the document for [path setting of sumo](https://sumo.dlr.de/docs/Basics/Basic_Computer_Skills.html#configuring_path_settings).